doc/source/math_num_documentation/forward_structure.rst

```diff
@@ -493,8 +493,112 @@ Hydrological processes can be described at pixel scale in `smash` with one of th
 
     .. math::
 
-        q_t(x, t) = q_r(x, t) + q_{e}(x, t) + q_d(x, t) 
+        q_t(x, t) = q_r(x, t) + q_{e}(x, t) + q_d(x, t)
 
+.. _math_num_documentation.forward_structure.hydrological_module.grc:
+
+.. dropdown:: grc (Génie Rural C)
+    :animate: fade-in-slide-down
+
+    This hydrological operator is derived from the GR models. It consists in a ``gr4`` like model stucture
+    with a second [slow-]transfer reservoir. This structure is used for the Reunion island.
+
+    .. figure:: ../_static/grc_structure.svg
+        :align: center
+        :width: 300
+        
+        Diagram of the ``grc`` hydrological operator
+
+    It can be expressed as follows:
+
+    .. math::
+
+        q_{t}(x, t) = f\left(\left[P, E\right](x, t), m_{lt}(x, t), \left[c_i, c_p, c_t, c_l, k_{exc}\right](x), \left[h_i, h_p, h_t, h_l\right](x, t)\right)
+
+    with :math:`q_{t}` the elemental discharge, :math:`P` the precipitation, :math:`E` the potential evapotranspiration,
+    :math:`m_{lt}` the melt flux from the snow operator, :math:`c_i` the maximum capacity of the interception reservoir,
+    :math:`c_p` the maximum capacity of the production reservoir, :math:`c_t` the maximum capacity of the transfer reservoir,
+    :math:`c_l` the maximum capacity of the [slow-]transfer reservoir, :math:`k_{exc}` the exchange coefficient,
+    :math:`h_i` the state of the interception reservoir, :math:`h_p` the state of the production reservoir,
+    :math:`h_t` the state of the transfer reservoir and :math:`h_t` the state of the [slow-]transfer reservoir.
+
+    .. note::
+
+        Linking with the forward problem equation :ref:`Eq. 1 <math_num_documentation.forward_inverse_problem.forward_problem_M_1>`
+        
+        - Internal fluxes, :math:`\{q_{t}, m_{lt}\}\in\boldsymbol{q}`
+        - Atmospheric forcings, :math:`\{P, E\}\in\boldsymbol{\mathcal{I}}`
+        - Parameters, :math:`\{c_i, c_p, c_t, c_l, k_{exc}\}\in\boldsymbol{\theta}`
+        - States, :math:`\{h_i, h_p, h_t, h_l\}\in\boldsymbol{h}`
+
+    The function :math:`f` is resolved numerically as follows:
+
+    **Interception**
+
+    Same as ``gr4`` interception, see :ref:`GR4 Interception <math_num_documentation.forward_structure.hydrological_module.gr4>`
+
+    **Production**
+
+    Same as ``gr4`` production, see :ref:`GR4 Production <math_num_documentation.forward_structure.hydrological_module.gr4>`
+
+    **Exchange**
+
+    Same as ``gr4`` exchange, see :ref:`GR4 Exchange <math_num_documentation.forward_structure.hydrological_module.gr4>`
+
+    **Transfer**
+
+    - Split the production runoff :math:`p_r` into three branches (transfer, [slow-]transfer and direct), :math:`p_{rr}`, :math:`p_{rl}` and :math:`p_{rd}`
```

Comment:
delayed and direct ?

Reply:
je suis partant pour mettre "delayed transfer" et "direct transfer"